spec/workers/users/resave_articles_worker_spec.rb

```diff
@@ -7,17 +7,17 @@
     let(:worker) { subject }
 
     context "with user" do
-      it "resave articles" do
-        user = create(:user)
-        article = create(:article, user: user)
+      let_it_be(:user) { create(:user) }
+      let_it_be(:article) { create(:article, user: user) }
 
+      it "resave articles" do
         old_updated_at = article.updated_at
 
-        Timecop.travel(1.minute.from_now) do
```

Comment:
Technically I think this part didn't have much to do with parallel builds and maybe could stay?

Reply:
yeah, let me apply that